sonosco/datasets/download_datasets/data_utils.py

```diff
@@ -0,0 +1,44 @@
+import fnmatch
+import io
+import os
+import logging
+import torch.distributed as dist
+import sonosco.common.audio_tools as audio_tools
+
+from tqdm import tqdm
+
+LOGGER = logging.getLogger(__name__)
+
+
+def create_manifest(data_path, output_path, min_duration=None, max_duration=None):
```

Comment:
It's the code from deepspeech that you also had. What is so shitty about it? It does its job.

Reply:
It can probably be rewritten to look nicer, no doubt about that. And in general regarding functions which are not part of a class I think in python it's a good thing. Because these aren't just global functions flying around, they are part of the module which already groups them logically. And having all functions as part of a class is just boilerplate code in my opinion.